tests/func/test_update.py

```diff
@@ -172,3 +172,35 @@ def test_update_import_url(tmp_dir, dvc, tmp_path_factory):
 
     assert dst.is_file()
     assert dst.read_text() == "updated file content"
+
+
+def test_update_git_tracked(tmp_dir, dvc, erepo_dir):
+    with erepo_dir.chdir():
+        erepo_dir.scm.repo.index.remove([".dvc"], r=True)
+        shutil.rmtree(".dvc")
+        erepo_dir.scm_gen("file", "first version")
+        erepo_dir.scm.add(["file"])
+        erepo_dir.scm.commit("first version")
+
+    stage = dvc.imp(fspath(erepo_dir), "file", "file")
+
+    # Just to make sure it doesn't crash
+    dvc.update(stage.path)
```

Comment:
why would this crash?

Reply:
probably defensive testing, to check that there's no error even if the file is unchanged?

Reply:
That's right. The purpose here is to increase coverage a bit.